src/Kafka/ConsumerGroups/components/ConsumerGroupDetail.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  Flex,
+  FlexItem,
+  Stack,
+  TextContent,
+  TextVariants,
+  Text,
+} from "@patternfly/react-core";
+import {
+  TableComposable,
+  TableVariant,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@patternfly/react-table";
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { ConsumerGroupPopover } from "./ConsumerGroupPopover";
+import { ConsumerGroup } from "../types";
+
+export type ConsumerGroupDetailProps = {
+  consumerGroupByTopic: boolean;
+  consumerGroupDetail: ConsumerGroup;
+};
+
+export const ConsumerGroupDetail: FunctionComponent<
+  ConsumerGroupDetailProps
+> = ({ consumerGroupByTopic, consumerGroupDetail }) => {
+  const { t } = useTranslation(["kafka"]);
+
+  const columnNames = {
+    topic: t("topic.topic"),
+    partition: t("consumerGroup.partition"),
+    consumer_id: t("consumerGroup.consumer_id"),
+    current_offset: t("consumerGroup.current_offset"),
+    log_end_offset: t("consumerGroup.log_end_offset"),
+    offset_lag: t("consumerGroup.offset_lag"),
+  };
+
+  return (
+    <Stack hasGutter>
+      <TextContent>
+        <Flex>
+          <FlexItem>
+            <Text component={TextVariants.h4} size={50}>
+              {consumerGroupByTopic
+                ? t("consumerGroup.active_members_for_topic")
+                : t("consumerGroup.active_members")}{" "}
+            </Text>
+            <div>
+              <Text component={TextVariants.h2}>
+                {consumerGroupDetail &&
+                  consumerGroupDetail.consumers.reduce(function (prev, cur) {
+                    return prev + (cur.partition != -1 ? 1 : 0);
+                  }, 0)}
+              </Text>
+            </div>
+          </FlexItem>
+          <FlexItem>
+            <Text component={TextVariants.h4}>
+              {consumerGroupByTopic
+                ? t("consumerGroup.partitions_with_lag_for_topic")
+                : t("consumerGroup.partitions_with_lag")}{" "}
+              <ConsumerGroupPopover
+                title={t("consumerGroup.partitions_with_lag_name")}
+                description={t("consumerGroup.partitions_with_lag_description")}
+              />
+            </Text>
+            <div>
```

Comment:
@jgiardino , @riccardo-forina earlier instead of <div>,  <Text component={TextVariants.p}> was used but the test was failing with the below error so Changed to div, please suggest me if it's a good approach 
![Uploading Screenshot from 2022-03-04 13-35-55.png…]()

Reply:
I think you can remove the wrapper completely. The error is right, you can't have an h2 inside a p, but we just have an h2 so I don't see the need to wrap it into a div